### PR TITLE
Config docs and README refresh

### PR DESCRIPTION
**Pull request by lornajane**
## What/Why/How?

Configuring Redoc can be confusing, partly because we use our central configuration docs, with some-but-not-all options supported in Redoc. Also the README on this project is _epic_. This pull request slims down the README, creates a dedicated page for configuration options, and also a docs landing page in case anyone finds the project that way on our main docs.

## Check yourself

- [x] Code is linted
- [x] Tested
- [x] All new/updated code is covered with tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
 To test it with your own OpenAPI definition,
 enter the URL for your definition and select **TRY IT**.
 
-## Redoc vs. Reference vs. Portals
+## Redoc Features
```
Comment (HCloward) [inline README.md:33]:
```suggestion
## Redoc features
```

File: README.md [hunk 2]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-
-- [Portals](https://redocly.com/docs/developer-portal/introduction/)
-- [Reference](https://redocly.com/docs/api-reference-docs/getting-started/)
-- [Redoc](https://redocly.com/docs/redoc/quickstart/intro/)
-
-## Features
 - Responsive three-panel design with menu/scrolling synchronization
-- [Multiple deployment options](https://redocly.com/docs/redoc/quickstart/intro/)
-- [Server-side rendering (SSR) ready](https://redocly.com/docs/redoc/quickstart/cli/#redoc-cli-commands)
+- Support for OpenAPI 3.1, OpenAPI 3.0 and Swagger 2.0
```
Comment (HCloward) [inline README.md:36]:
```suggestion
- Support for OpenAPI 3.1, OpenAPI 3.0, and Swagger 2.0
```

File: README.md [hunk 3]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-[<img alt="Customization services" src="http://i.imgur.com/c4sUF7M.png" height="60px">](https://redocly.com/#services)
-- High-level grouping in side-menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
-- Branding/customizations using the [`theme` option](https://redocly.com/docs/api-reference-docs/configuration/theming/)
-
-## Support
-- OpenAPI v3.0 support
-- Basic OpenAPI v3.1 support
-- Broad OpenAPI v2.0 feature support (yes, it supports even `discriminator`) <br>
-  ![](docs/images/discriminator-demo.gif)
 - Code samples support (via vendor extension) <br>
```
Comment (HCloward) [inline README.md:41]:
```suggestion
- Code samples support (with vendor extension) <br>
```

File: README.md [hunk 4]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-OpenAPI definition before publishing.
+```
+npx @redocly/cli build-docs openapi.yaml 
+```
 
-Learn more about [API standards and linting](https://redocly.com/docs/cli/api-standards/) in the main Redocly documentation.
+The tool outputs by default to a file named `redoc-static.html` that you can open in your browser.
 
-## Deployment
+> [Redocly CLI](https://github.com/Redocly/redocly-cli/) does more than docs, check it out and add linting, bundling and more to your API workflow.
```
Comment (HCloward) [inline README.md:58]:
```suggestion
> [Redocly CLI](https://github.com/Redocly/redocly-cli/) does more than docs, check it out and add linting, bundling, and more to your API workflow.
```

File: README.md [hunk 5]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
 
-### TL;DR final code example
+### Add an HTML element to the page
 
-To render your OpenAPI definition using Redoc, use the following HTML code sample and
-replace the `spec-url` attribute with the url or local file address to your definition.
+Create an HTML page, or edit an existing one, and add the following:
```
Comment (HCloward) [inline README.md:62]:
```suggestion
Create an HTML page, or edit an existing one, and add the following within the body tags:
```

File: README.md [hunk 6]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-</html>
-
 ```
 
-For step-by-step instructions for how to get started using Redoc
-to render your OpenAPI definition, refer to the
-[**Redoc quickstart guide**](https://redocly.com/docs/redoc/quickstart/) and [**How to use the HTML element**](https://redocly.com/docs/redoc/deployment/html/).
+Open the HTML file in your browser, and your API documentation is shown on the page.
+
+Add your own `spec-url` to the `<redoc>` tag, this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
```
Comment (HCloward) [inline README.md:71]:
```suggestion
Add your own `spec-url` to the `<redoc>` tag; this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
```

File: README.md [hunk 7]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
+### Documentation and resources
+
+- [Reference docs](https://redocly.com/docs/api-reference-docs/getting-started/) - we take care of the hosting
+- [Redoc](https://redocly.com/docs/redoc/) - detailed documentation for this open source project (also in the `docs/` folder)
+- [Command-line interface to bundle your docs into a web-ready HTML file](https://redocly.com/docs/cli/commands/build-docs/)
+- API linting, bundling, and much more with open source [Redocly CLI](https://redocly.com/docs/cli)
+
+## Showcase
+
+A sample of the organisations using Redocly tools in the wild:
```
Comment (HCloward) [inline README.md:97]:
```suggestion
A sample of the organizations using Redocly tools in the wild:
```

File: README.md [hunk 8]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-### Security Definition location
-You can inject the Security Definitions widget into any place in your definition `description`.
-For more information, refer to [Security definitions injection](docs/security-definitions-injection.md).
+Redoc is highly configurable, see the [configuration documentation](docs/config.md) for details.
 
 ### OpenAPI specification extensions
 Redoc uses the following [specification extensions](https://redocly.com/docs/api-reference-docs/spec-extensions/):
+
 * [`x-logo`](docs/redoc-vendor-extensions.md#x-logo) - is used to specify API logo
 * [`x-traitTag`](docs/redoc-vendor-extensions.md#x-traitTag) - useful for handling out common things like Pagination, Rate-Limits, etc
```
Comment (HCloward) [inline README.md:117]: I'm not sure about this wording here - "handling out" - not sure what it means

File: README.md [hunk 9]
```diff
@@ -199,126 +127,19 @@ Redoc uses the following [specification extensions](https://redocly.com/docs/api
 * [`x-extendedDiscriminator`](docs/redoc-vendor-extensions.md#x-extendedDiscriminator) - In Schemas, uses this to solve name-clash issues with the standard discriminator
```
Comment (HCloward) [inline README.md:127]: All bullets should be capitalized or none but having these two start with capitalized words when the others do not is inconsistent

File: docs/config.md [hunk 10]
```diff
@@ -0,0 +1,266 @@
+# Configure Redoc
+
+Getting your documentation just right is important, and Redoc comes with many configuration options to help you succeed in that mission.
+
+Each deployment type has documentation on how to options in its context. This page lists all the options you can use with Redoc.
```
Comment (HCloward) [inline docs/config.md:5]: This sentence is awkward. Do you mean how-to options? In the context of the deployment type?

File: docs/config.md [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+
+_Default: 3_
+
+### expandDefaultServerVariables
+
+Enables or disables expanding default server variables.
+
+### expandResponses
+
+Controls which responses to expand by default. Specify one or more responses by providing their response codes as a comma-separated list without spaces, e.g. `expandResponses='200,201'`. Special value 'all' expands all responses by default. Be careful: this option can slow down documentation rendering time.
```
Comment (HCloward) [inline docs/config.md:35]:
```suggestion
Controls which responses to expand by default. Specify one or more responses by providing their response codes as a comma-separated list without spaces, for example `expandResponses='200,201'`. Special value 'all' expands all responses by default. Be careful: this option can slow down documentation rendering time.
```

File: docs/config.md [hunk 12]
```diff
@@ -0,0 +1,266 @@
...
+
+### expandSingleSchemaField
+
+Automatically expands the single field in a schema.
+
+### hideDownloadButton
+
+Hides the 'Download' button for saving the API definition source file. **This does not make the API definition private**, it just hides the button.
```
Comment (HCloward) [inline docs/config.md:43]:
```suggestion
Hides the 'Download' button for saving the API definition source file. **This setting does not make the API definition private**; it just hides the button.
```

File: docs/config.md [hunk 13]
```diff
@@ -0,0 +1,266 @@
...
+
+### menuToggle
+
+If set to `true`, selecting an expanded item in the sidebar twice collapses it.
+
+_Default: true_
+
+### nativeScrollbars
+
+If set to `true`, the sidebar uses the native scrollbar instead of perfect-scroll. This is a scrolling performance optimization for big API definitions.
```
Comment (HCloward) [inline docs/config.md:99]:
```suggestion
If set to `true`, the sidebar uses the native scrollbar instead of perfect-scroll. This setting is a scrolling performance optimization for big API definitions.
```

File: docs/config.md [hunk 14]
```diff
@@ -0,0 +1,266 @@
...
+
+Shows the path link and HTTP verb in the middle panel instead of the right panel.
+
+### payloadSampleIdx
+
+If set, the payload sample is inserted at the specified index. If there are `N` payload samples and the value configured here is bigger than `N`, the payload sample is inserted last. Indexes start from 0.
+
+### requiredPropsFirst
+
+Shows required properties in schemas first, ordered in the same order as in required array.
```
Comment (HCloward) [inline docs/config.md:115]:
```suggestion
Shows required properties in schemas first, ordered in the same order as in the required array.
```

File: docs/config.md [hunk 15]
```diff
@@ -0,0 +1,266 @@
...
+
+### schemaExpansionLevel
+
+Specifies whether to automatically expand schemas in Reference docs. Set it to `all` to expand all schemas regardless of their level, or set it to a number to expand schemas up to the specified level. For example, `schemaExpansionLevel: 3` expands schemas up to three levels deep. The default value is `0`, meaning no schemas are expanded automatically.
+
+### scrollYOffset
+
+Specifies a vertical scroll-offset.
+This is useful when there are fixed positioned elements at the top of the page, such as navbars, headers etc.
```
Comment (HCloward) [inline docs/config.md:124]:
```suggestion
This setting is useful when there are fixed positioned elements at the top of the page, such as navbars, headers, etc.
```

File: docs/config.md [hunk 16]
```diff
@@ -0,0 +1,266 @@
...
+- as a CSS selector - the selector of the element to be used for specifying the offset. The distance from the top of the page to the element's bottom is used as the offset.
+- a function (advanced) - a getter function. Must return a number representing the offset (in pixels).
+
+### showExtensions
+
+Shows specification extensions ('x-' fields). Extensions used by Redoc are ignored. The value can be boolean or an array of strings with names of extensions to display. When used as boolean and set to `true`, all specification extensions are shown.
+
+### showObjectSchemaExamples
+
+Show object schema example in the properties, default false.
```
Comment (HCloward) [inline docs/config.md:137]:
```suggestion
Shows object schema example in the properties; default `false`.
```

File: docs/config.md [hunk 17]
```diff
@@ -0,0 +1,266 @@
...
+
+When set to `true`, sorts operations in the navigation sidebar and in the middle panel alphabetically.
+
+### sortPropsAlphabetically
+
+When set to `true`, sorts properties in all schemas alphabetically.
+
+### sortTagsAlphabetically
+
+When set to true, sorts tags in the navigation sidebar and in the middle panel alphabetically. Note that only tags are sorted alphabetically in the middle panel, not the operations associated with each tag. To sort operations alphabetically as well, you must set sortOperationsAlphabetically to true.
```
Comment (HCloward) [inline docs/config.md:161]:
```suggestion
When set to true, sorts tags in the navigation sidebar and in the middle panel alphabetically. Note that only tags are sorted alphabetically in the middle panel, not the operations associated with each tag. To sort operations alphabetically as well, you must set the `sortOperationsAlphabetically` setting to `true`.
```

File: docs/index.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
+---
+title: Redoc
+---
+
+# Redoc: Open source API documentation tool
+
+Redoc is a clean and easy way to produce web-ready documentation from an OpenAPI description (Swagger is also still supported). With one command, create your documentation, and customise it to meet the needs of your users.
```
Comment (HCloward) [inline docs/index.md:7]:
```suggestion
Redoc is a clean and easy way to produce web-ready documentation from an OpenAPI description (Swagger is also still supported). With one command, create your documentation, and customize it to meet the needs of your users.
```

File: docs/index.md [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+Create an HTML page, or edit an existing one, and add the following:
+
+```html
+    <redoc spec-url="http://petstore.swagger.io/v2/swagger.json"></redoc>
+    <script src="https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js"> </script>
+```
+
+Open the HTML file in your browser, and your API documentation is shown on the page.
+
+Add your own `spec-url` to the `<redoc>` tag, this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
```
Comment (HCloward) [inline docs/index.md:49]:
```suggestion
Add your own `spec-url` to the `<redoc>` tag; this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
```

File: docs/index.md [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+### More usage options
+
+Check out the [deployment documentation](./deploment/index/md) for more options, and detailed documentation for each.
+
+## Configure Redoc
+
+Redoc is highly configurable. Each deployment option accepts configuration in a way that's appropriate to that platform, but the options are the same for each. For example:
+
+* Using [Redocly CLI](../cli/index.md), configuration goes in the `redocly.yaml` file, or can be supplied as command line parameters, such as `--theme.openapi.disableSearch`.
+* Using HTML or React or react, configure by setting `option` in the tag.
```
Comment (HCloward) [inline docs/index.md:60]:
```suggestion
* Using HTML or React, configure by setting `option` in the tag.
```

File: docs/index.md [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+      colors:
+        primary:
+          main: '#660099'
+      typography:
+        fontSize: 14pt
+        headings:
+          fontWeight: bold
+```
+
+Redocly CLI detects a file named `redocly.yaml` in the same directory as you run the command and uses it. See the documentation with a command like this:
```
Comment (HCloward) [inline docs/index.md:84]:
```suggestion
Redocly CLI detects a file named `redocly.yaml` in the same directory as you run the command and use it. See the documentation with a command like this:
```

File: docs/index.md [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+```
+redocly preview-docs openapi.yaml
+```
+
+There are many, many more options available. Visit the [configuration reference](./config.md) for a complete list.
+
+## Next steps
+
+* If you are new to OpenAPI, try the [OpenAPI starter project](../../cli/openapi-starter/) for a great introduction.
+* Ready to build documentation from an existing OpenAPI file? The [Redoc quickstart](./quickstart.md).
```
Comment (HCloward) [inline docs/index.md:95]:
```suggestion
* Ready to build documentation from an existing OpenAPI file? Go to the [Redoc quickstart](./quickstart.md) and get started.
```

File: README.md [hunk 23]
```diff
@@ -3,17 +3,15 @@
...
+  [![npm](http://img.shields.io/npm/v/redoc.svg)](https://www.npmjs.com/package/redoc) [![License](https://img.shields.io/npm/l/redoc.svg)](https://github.com/Redocly/redoc/blob/main/LICENSE)
 
-  [![bundle size](http://img.badgesize.io/https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js?compression=gzip&max=300000)](https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js) [![npm](https://img.shields.io/npm/dm/redoc.svg)](https://www.npmjs.com/package/redoc) [![](https://data.jsdelivr.com/v1/package/npm/redoc/badge)](https://www.jsdelivr.com/package/npm/redoc) [![Docker Build Status](https://img.shields.io/docker/build/redocly/redoc.svg)](https://hub.docker.com/r/redocly/redoc/)
+  [![npm](https://img.shields.io/npm/dm/redoc.svg)](https://www.npmjs.com/package/redoc) [![](https://data.jsdelivr.com/v1/package/npm/redoc/badge)](https://www.jsdelivr.com/package/npm/redoc)
```
Comment (RomanHotsiy) [inline README.md:8]: I would keep the bundle size. This is one of the main things I personally look for this when evaluating npm packages.

File: README.md [hunk 24]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-
-## Customization options
-[<img alt="Customization services" src="http://i.imgur.com/c4sUF7M.png" height="60px">](https://redocly.com/#services)
-- High-level grouping in side-menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
-- Branding/customizations using the [`theme` option](https://redocly.com/docs/api-reference-docs/configuration/theming/)
-
-## Support
-- OpenAPI v3.0 support
-- Basic OpenAPI v3.1 support
-- Broad OpenAPI v2.0 feature support (yes, it supports even `discriminator`) <br>
```
Comment (RomanHotsiy) [inline README.md:99]: I would move this gif into the features somehow.

File: README.md [hunk 25]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-- [Server-side rendering (SSR) ready](https://redocly.com/docs/redoc/quickstart/cli/#redoc-cli-commands)
+- Support for OpenAPI 3.1, OpenAPI 3.0 and Swagger 2.0
+- [Multiple deployment options](https://redocly.com/docs/redoc/)
+- High-level grouping in side menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
 - Ability to integrate your API introduction into the side menu
 - [Simple integration with `create-react-app`](https://redocly.com/docs/redoc/quickstart/react/)
-
-  [Example repo](https://github.com/APIs-guru/create-react-app-redoc)
-- [Command-line interface to bundle your docs into a **zero-dependency** HTML file](https://redocly.com/docs/cli/commands/build-docs/)
-- Neat **interactive** documentation for nested objects <br>
```
Comment (RomanHotsiy) [inline README.md:88]: I don't like the wording here but I would keep the gif and the feature.

File: README.md [hunk 26]
```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
...
-    <style>
-      body {
-        margin: 0;
-        padding: 0;
-      }
-    </style>
-  </head>
-  <body>
-    <redoc spec-url='http://petstore.swagger.io/v2/swagger.json'></redoc>
+    <redoc spec-url="http://petstore.swagger.io/v2/swagger.json"></redoc>
```
Comment (RomanHotsiy) [inline README.md:65]: I would keep the whole html here. It may be simpler for some people to understand.

**Review by RomanHotsiy — COMMENTED** [review]
It looks great!

I left a few comments.

File: README.md (hunk 25 above)
Comment (lornajane) [reply README.md:88]: gifs in READMEs are so wrong and we have so many of them. I'll put the text back.

File: README.md (hunk 24 above)
Comment (lornajane) [reply README.md:99]: No more gifs! Especially not related to OpenAPI 2.0.

File: README.md (hunk 26 above)
Comment (lornajane) [reply README.md:65]: Would it work to put the full example on the deployment page? This snippet should be enough for quick reference, but the whole example does need to exist somewhere easy to find. Although currently the HTML install instructions are like this https://redocly.com/docs/redoc/deployment/html/ which seems like it refers to something different. I think I have more to do here!

File: docs/config.md (hunk 10 above)
Comment (lornajane) [reply docs/config.md:5]: I think it's missing a word :laughing: Let me try again ...

File: docs/index.md (hunk 21 above)
Comment (lornajane) [reply docs/index.md:84]: Confused about this one. The CLI detects a file, then uses it.

File: README.md (hunk 8 above)
Comment (lornajane) [reply README.md]: I updated it, it's basically to revert the tags to being used as just tags, not as navigation.

File: README.md (hunk 9 above)
Comment (lornajane) [reply README.md]: Fixed!

Comment (lornajane) [reply README.md]: Especially for a documentarian-targeted tool, this is a really good spot, thank you

File: docs/config.md (hunk 10 above)
Comment (lornajane) [reply docs/config.md]: Is that better?

File: README.md (hunk 23 above)
Comment (lornajane) [reply README.md]: No problem! I've put it back

File: README.md [hunk 27]
```diff
@@ -32,161 +30,92 @@ A version of the Swagger Petstore API is displayed by default.
...
-OpenAPI definition before publishing.
+```
+npx @redocly/cli build-docs openapi.yaml 
+```
 
-Learn more about [API standards and linting](https://redocly.com/docs/cli/api-standards/) in the main Redocly documentation.
+The tool outputs by default to a file named `redoc-static.html` that you can open in your browser.
 
-## Deployment
+> [Redocly CLI](https://github.com/Redocly/redocly-cli/) does more than docs, check it out and add linting, bundling, and more to your API workflow.
```
Comment (HCloward) [inline README.md:59]:
```suggestion
> [Redocly CLI](https://github.com/Redocly/redocly-cli/) does more than docs; check it out and add linting, bundling, and more to your API workflow.
```

File: docs/config.md [hunk 28]
```diff
@@ -0,0 +1,266 @@
...
+
+If set to `true`, the API definition is considered untrusted and all HTML/Markdown is sanitized to prevent XSS.
+
+## Theme settings
+
+* `spacing`
+  * `unit`: 5 # main spacing unit used in autocomputed theme values later
+  * `sectionHorizontal`: 40 # Horizontal section padding. COMPUTED: spacing.unit * 8
+  * `sectionVertical`: 40 # Horizontal section padding. COMPUTED: spacing.unit * 8
+* `breakpoints` # breakpoints for switching three/two and mobile view layouts
```
Comment (HCloward) [inline docs/config.md:175]:
```suggestion
* `breakpoints`: # breakpoints for switching three/two and mobile view layouts
```

File: docs/config.md [hunk 29]
```diff
@@ -0,0 +1,266 @@
...
+    * `visited`: # COMPUTED: typography.links.color
+    * `hover`: # COMPUTED: lighten(0.2 typography.links.color)
+    * `textDecoration`: 'auto'
+    * `hoverTextDecoration`: 'auto'
+* `sidebar`
+  * `width`: '260px'
+  * `backgroundColor`: '#fafafa'
+  * `textColor`: '#333333'
+  * `activeTextColor`: # COMPUTED: theme.sidebar.textColor (if set by user) or theme.colors.primary.main
+  * `groupItems` # Group headings
```
Comment (HCloward) [inline docs/config.md:213]:
```suggestion
  * `groupItems`: # Group headings
```

File: docs/index.md (hunk 21 above)
Comment (HCloward) [reply docs/index.md:84]: ok, I see what you mean - never mind

**Review by HCloward — APPROVED** [review]
These updates look great!

File: README.md (hunk 26 above)
Comment (RomanHotsiy) [reply README.md:66]: It would work.

File: README.md (hunk 25 above)
Comment (RomanHotsiy) [reply README.md:88]: what's wrong with gifs in README? I see them very often (sometimes videos) and find them useful but I must be missing something 🤔

Comment (lornajane) [reply README.md:88]: They're common, but they're inacessible, distracting or seizure-inducing, and if you can't explain what a feature is with words, do you even know what feature you've built? Especially for a developer tool, if the README isn't designed to be read, what else did you not understand about tools and developers??

(I know this is controversial, but really. I don't think it gives the right image at all when it's in the README - but we should totally link to great video content on these features)

File: docs/config.md (hunk 28 above)
Comment (lornajane) [reply docs/config.md:175]: None of the other top-level entries have colons, so I'm not adding this one.

File: docs/config.md (hunk 29 above)
Comment (lornajane) [reply docs/config.md:213]: It looks like we don't have colons where there are sub-entries. I'd like to overhaul this part completely, but it's sort of internally consistent so I'm leaving it as-is for now.

**Comment by lornajane** [comment]
@AlexVarchuk could you please check over these changes? I've completely rewritten the HTML deployment page, removed the old quickstart and directed people to the HTML page as our easiest onboarding. If you are happy that everything looks accurate, then I think we are probably good to go. Thank you!

**Comment by AlexVarchuk** [comment]
@lornajane It looks good to me! Thank you! 🙌

**Review by AlexVarchuk — APPROVED** [review]

File: docs/deployment/html.md [hunk 30]
```diff
@@ -1,123 +1,144 @@
...
+  <body>
+    <redoc spec-url='http://petstore.swagger.io/v2/swagger.json'></redoc>
+    <script src="https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js"> </script>
+  </body>
+</html>
+```
 
-- [npm](https://docs.npmjs.com/about-npm)
-- [yarn](https://classic.yarnpkg.com/en/docs/getting-started)
+:::Success URL or local file
```
Comment (adamaltman) [inline docs/deployment/html.md:41]: Needs a lowercase `s` in success.

File: docs/index.md [hunk 31]
```diff
@@ -0,0 +1,96 @@
...
+    <script src="https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js"> </script>
+```
+
+Open the HTML file in your browser, and your API documentation is shown on the page.
+
+Add your own `spec-url` to the `<redoc>` tag; this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
+
+### More usage options
+
+Check out the [deployment documentation](./deploment/index/md) for more options, and detailed documentation for each.
```
Comment (adamaltman) [inline docs/index.md:53]: This link looks wrong.

**Comment by lornajane** [comment]
Thanks! and Yep! Without preview tooling, we missed things when I was writing and in review. I fixed at least one of these on the related sidebars PR on the marketing website.
